### ISSUE TITLE
Is there a way to get properities from the TLS sessions?

### ISSUE THREAD

**Issue by jimsch** [chkr1011/MQTTnet#658]
For security purposes, I want to get the tls-unique value from TLS connection?  I am looking to either get the tls-unique value or some other value from the exporter so that it can be signed for security purposes.

**Comment by chkr1011** [comment]
With 3.0.3 the client certificate is exposed to the connection validator. Is that solving your issue?

**Comment by jimsch** [comment]
No.  This is connection specific material and is not part of the certificate.

**Comment by jimsch** [comment]
No.  This is connection specific material and is not part of the certificate.  This is on HttpListener as of Net 4.6.1

**Comment by chkr1011** [comment]
Sorry but this data is currently not available.

**Comment by JanEggers** [comment]
@jimsch you could have a look at https://github.com/chkr1011/MQTTnet/blob/2918be18c6c6529584cd1d78e09ebb3fc0819e4f/Source/MQTTnet.AspnetCore/

you maybe can create your own MqttConnectionContext/MqttConnectionHandler  and expose / verify the details you need.
